griptape/artifacts/video_artifact.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from attrs import define, field
+
+from griptape.artifacts import BlobArtifact
+
+
+@define
+class VideoArtifact(BlobArtifact):
+    """Stores video binary data and relevant metadata.
+
+    Attributes:
+        value: The video binary data.
+        mime_type: The video MIME type.
+        resolution: The resolution of the video (e.g., 1920x1080).
+        duration: Duration of the video in seconds.
+    """
+
+    aspect_ratio: tuple[int, int] = field(default=(16, 9), kw_only=True)
+
+    @property
+    def mime_type(self) -> str:
+        return "video/mp4"  # Or make this flexible based on the video format
```

Comment:
sounds like our loaders support .ogg and .webm, can we point to the same loc for both or is there a reason to keep them separate?

Reply:
We should already have the same mime-type umbrella problem for image/audio, if its not in the framework already, check with collin on the preferred approach.

Also, should this be an actual attribute instead of a `@property`?

Reply:
`@property` is correct, just needs to be updated to something like [this](https://github.com/griptape-ai/griptape/blob/53bc38b5162ef927127a7f838d2c74c88cc0919d/griptape/artifacts/audio_artifact.py#L18-L20).